.cirrus.yml

```diff
@@ -0,0 +1,54 @@
+check_android_task:
+  name: Run Android instrumented tests
+  env:
+    SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspressample/files/screenshots
+    TUTORIAL_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
+    ALLURE_SAMPLE_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
```

Comment:
Seems like `TUTORIAL_SCREENSHOTS_ROOT_DIR` value is copy-pasted here :)

Reply:
Caught me